run-ws.py

```diff
@@ -0,0 +1,128 @@
+import os
+import json
+import socket
+import signal
+import asyncio
+import logging
+import argparse
+import websockets
+
+from jsonschema import validate
+from chainlink import Chainlink
+
+import grader.api_keys as api_keys
+from grader.definitions import GRADING_JOB_DEF
+
+from config import *
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
+
+
+async def exec_job(job):
+    job_id = job[api_keys.GRADING_JOB_ID]
+    stages = job[api_keys.STAGES]
+
+    logger.info("starting job {}".format(job_id))
+
+    # execute job
+    try:
+        chain = Chainlink(stages, workdir=os.getcwd())
+        job_results = await chain.run_async({})
+    except Exception as ex:
+        logger.critical("grading job failed with exception:\n{}", ex)
+        job_results = [
+            {
+                "logs": {
+                    "stdout": b"the container crashed",
+                    "stderr": bytes(str(ex), "utf-8"),
+                },
+                "success": False,
+            }
+        ]
+
+    job_stdout = "\n".join(
+        [r["logs"]["stdout"].decode("utf-8") for r in job_results]
+    )
+    job_stderr = "\n".join(
+        [r["logs"]["stderr"].decode("utf-8") for r in job_results]
+    )
+
+    for r in job_results:
+        del r["logs"]
+
+    logger.info("finished job {}".format(job_id))
+
+    logger.info("job stdout:\n" + job_stdout)
```

Comment:
I think we should we using the `VERBOSE` flag present in the config for these two logs because they can be pretty heavy.

```
if VERBOSE:
    logger.info("job stdout:\n" + job_stdout)
    logger.info("job stderr:\n" + job_stderr)
```